library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.html

```diff
@@ -84,14 +84,14 @@ <h2 class="mat-title margin-bottom-xs" i18n>Pinned</h2>
       <!-- <mat-icon>settings</mat-icon> -->
       <span i18n>'Get Started' tour</span>
     </a>
-    <a mat-menu-item role="link" (click)="redirectToMyLearning()" class="app-features flex items-center">
+    <a mat-menu-item role="link" (click)="redirectToMyLearning();" class="app-features flex items-center">
```

Comment:
@garima468 Please remove the semicolon else build will fail

Reply:
removed